### PR TITLE
[RHPAM-2401-1] - QE review, db is not ba component 

### PR DESCRIPTION
**Pull request by spolti**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: process-migration/image.yaml [hunk 1]
```diff
@@ -19,6 +19,12 @@ envs:
     - name: "SCRIPT_DEBUG"
       example: "true"
       description: "If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed."
+    - name: JBOSS_KIE_ADMIN_USER
```
Comment (sutaakar) [inline process-migration/image.yaml]: This seems like unrelated change.

Comment (spolti) [reply process-migration/image.yaml]: yes, needs rebase.
